lib/twofishes/result.rb

```diff
@@ -36,21 +24,21 @@ def lng
       feature.geometry.center.lng
     end
 
-    def coordinates
-      [lat, lng]
+    def country_code
+      feature.cc
     end
 
-    def parents
```

Comment:
Why it is removed? We are not the only ones who uses `twofishes` gem, and we should provide some backward compability.

`responseIncludes` parameter from Twofishes interface should be implemented (for example as configuration parameter):
> responseIncludes=[comma separated list of ResponseIncludes integers or enum names] - ie responseIncludes=PARENTS,ALL_NAMES - influences what is contained within the returned interpretations. Most of these make the response slightly slower.
EVERYTHING - includes everything listed below
PARENTS - include the parent features of a venue (ie NY state, US inside the interpretation for 'new york city')
ALL_NAMES - include all names, not just in english + lang
PARENT_ALL_NAMES - same, but for parents
WKB_GEOMETRY - include polygon boundary geometry in WKB format if available
WKT_GEOMETRY - include polygon boundary geometry in WKT format if available
WKB_GEOMETRY_SIMPLIFIED - include simplified polygon boundary geometry in WKB format if available
WKT_GEOMETRY_SIMPLIFIED - include simplified polygon boundary geometry in WKT format if available
S2_COVERING - include S2 covering of polygon boundary as a list of S2 cell ids if available
REVGEO_COVERAGE - for ll+radius revgeo queries, turns on computing the percentage overlap between the circle query and the feature polygon. slow-ish
DISPLAY_NAME - controls if we should fetch parents to construct a string like "New York, New York, US" for legacy reasons, this is automatically turned on for geocode queries for now. It's mainly here because reverse geocode clients often don't need it